### ISSUE TITLE
Install instructions only mention plugin directory

### ISSUE THREAD

**Issue by Andre601** [LuckPerms/LuckPermsWeb#449]
The "How to install" section mentions this in point 3:

> Locate the plugin's configuration file (usually found within /plugins/LuckPerms/) and adjust any settings to your liking.

This is obviously favouring the plugin version for Spigot, Bungee and Velocity. Afaik (Haven't used LP for mods) does the folder for the mod version differ?

This should probably be improved a bit.

**Comment by Andre601** [comment]
Seems to be fixed already in translations, but not yet on the website itself?

**Comment by Turbotailz** [comment]
Translations from crowdin refresh every 6 hours.

**Comment by Andre601** [comment]
Yeah. Point 3 is still telling you to configure stuff in plugins/LuckPerms/ which afaik isn't true for the Fabric and forge version since those are mods?
Not sure how Nukkit and Sponge work here tho. But either way, given that point 2 mentions plugins and mods folder but point 3 doesn't mention the possible locations for the platforms is this a bit confusing...
